tests/resources/nimbus-testbeds/vic-simple-cluster-with-multiple-nfs.rb

```diff
@@ -0,0 +1,122 @@
+oneGB = 1 * 1000 * 1000 # in KB
+ 
+$testbed = Proc.new do
+  {
+    "name" => "vic-simple-cluster",
+    "version" => 3,
+    "esx" => (0..2).map do | idx |
+      {
+        "name" => "esx.#{idx}",
+        "vc" => "vc.0",
+        "style" => "fullInstall",
+        "desiredPassword" => "e2eFunctionalTest",
+        "memory" => 8192, # 2x default
+        "disks" => [ 30 * oneGB, 30 * oneGB, 30 * oneGB],
```

Comment:
while we are here, might as well get rid of this.  Don't need three small disks, just make one 50GB one or so